content/en/getting_started/agent/_index.md

```diff
@@ -12,90 +12,219 @@ further_reading:
 
 ## Setup
 
-If you haven't already, create a [Datadog account][7].
+### Prerequisites
 
-### Installation
+1. Create a [Datadog account][14].
 
-The Agent can be installed on many different platforms either directly on the host or as a [containerized version][8]. Most systems have a one-line install option.
+2. You will need access to your [Datadog API key][15]. 
 
-{{< partial name="platforms/platforms.html" desc="Choose your platform to see installation instructions:" links="gs" >}}
+3. Have the Datadog UI open.
 
-### Configuration
+**Note** This walkthrough will be using the Ubuntu platform. See the [Basic Agent Usage][16] page for a full list of supported platforms. 
 
-The Agent's [main configuration file][9] is `datadog.yaml`. The required parameters are your [Datadog API key][10] which is used to associate your Agent's data with your organization and the Datadog site ({{< region-param key="dd_site" code="true" >}}). See the [sample config_template.yaml][11] for all available configuration options.
+### Installation
 
-For the [container Agent][8], `datadog.yaml` configuration options are passed in with [environment variables][12], for example:
+In the Datadog UI, go to the [Agent Installation Page][17] for Ubuntu, **Integrations > Agent** 
 
-- `DD_API_KEY` for the Datadog API key
-- `DD_SITE` for the Datadog site
+To install the Datadog Agent on a host, use the [one line install command][18] updated with your [Datadog API key][2]:
+
+```shell
+DD_API_KEY=<DATADOG_API_KEY> DD_SITE="{{< region-param key="dd_site" >}}" bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)"
+```
 
 ### Validation
 
-Run the Agent's [status command][13] to verify installation.
+#### Terminal Command
 
-### Commands
+Run the Agent's [status command][19] to verify installation.
 
-See [Agent Commands][14] to [Start][15], [Stop][16] or [Restart][17] your Agent.
+```shell
+sudo datadog-agent status
+```
+A successfull installation will return an Agent Status report that begins with Agent information like this:
 
-## Data Collected
+```shell
+===============
+Agent (v7.36.1)
+===============
 
-### Metrics
+  Status date: 2022-06-15 15:54:48.364 EDT / 2022-06-15 19:54:48.364 UTC (1655322888364)
+  Agent start: 2022-06-15 15:54:29.85 EDT / 2022-06-15 19:54:29.85 UTC (1655322869850)
+  Pid: 9801
+  Go Version: go1.17.6
+  Python Version: 3.8.11
+  Build arch: amd64
+  Agent flavor: agent
+  Check Runners: 6
+  Log Level: info
+```
 
-#### Agent
+#### Events
 
-The metrics below are available with Agent v6. For Agent v5, see the [Agent Metrics][18] integration.
+In the Datadog UI, go to the Events Explorer Page **Events > Explorer**. The Agent sends events to Datadog when an Agent is started or restarted. The following message will display if your Agent successfully installs:
```

Comment:
```suggestion
In the Datadog UI, go to the Events Explorer Page **Events > Explorer**. The Agent sends events to Datadog when an Agent is started or restarted. The following message displays if your Agent successfully installs:
```
We avoid using the future tense in the docs. Any time you have the word "will", see if you can rewrite it without.